### PR TITLE
Fix/ai-move-preferences

### PR DESCRIPTION
**Pull request by BWalusiak**
Working ai

### CODE REVIEW

File: app/src/main/java/com/cyberbot/checkers/game/Grid.java [hunk 1]
```diff
@@ -539,38 +539,40 @@ int getValue(PlayerNum playerNum, PlayerNum adversaryNum) {
...
+                value += Math.abs(gridEntry.getX()+1 - (size/2));
 
-                if (gridEntry.getPlayer() == PlayerNum.FIRST) {
-                    value += gridEntry.getY() / 2;
-                } else if (gridEntry.getPlayer() == PlayerNum.SECOND) {
-                    value += Math.abs((gridEntry.getY() - getSize()) / 2);
+                //Prioritize forward movement
+
+                if (playerNum == PlayerNum.FIRST){
+                    value += gridEntry.getY()+1;
```
Comment (RouNNdeL) [inline app/src/main/java/com/cyberbot/checkers/game/Grid.java]: Inconsistent space usage

File: app/src/main/java/com/cyberbot/checkers/game/Grid.java [hunk 2]
```diff
@@ -539,38 +539,40 @@ int getValue(PlayerNum playerNum, PlayerNum adversaryNum) {
...
+                value += 20;
+
+                if (gridEntry.getPieceType() == PieceType.KING) {
+                value += 80;
+                }
 
-            // Prioritize sides
-            value += Math.abs((gridEntry.getX()) + gridEntry.getY() - getSize()) / 4;
+                // Prioritize sides
+                value += Math.abs(gridEntry.getX()+1 - (size/2));
```
Comment (RouNNdeL) [inline app/src/main/java/com/cyberbot/checkers/game/Grid.java]: Inconsistent space usage

**Review by RouNNdeL — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/cyberbot/checkers/game/Grid.java (hunk 2 above)
Comment (BWalusiak) [reply app/src/main/java/com/cyberbot/checkers/game/Grid.java]: Done.

File: app/src/main/java/com/cyberbot/checkers/game/Grid.java (hunk 1 above)
Comment (BWalusiak) [reply app/src/main/java/com/cyberbot/checkers/game/Grid.java]: Done.

**Review by RouNNdeL — APPROVED** [review]

**Comment by RouNNdeL** [comment]
I'll allow this merge for now. But before the next release please revert to using `attemptMove`.

**Pull request merged**